docs/azure/services/Microsoft_Purview_Information_Protection_Overview.md

```diff
@@ -0,0 +1,96 @@
+- Assign permissions now or let users decide
+- User access expiration
+- Allow or disallow offline access - users accessing the file need to be connected to the internet
+- Assign permissions to specific groups - for example, all users in the HR group can edit and view the files, whilst every other internal user can only view file.
+
+#### Auto-labelling for files and emails
+
+This option provides a way to configure how the data will be discovered. As mentioned above, it's possible to discover it manually, using sensitive info type and trainable classifiers. If opted for SIT, patterns will be available for review in this configuration menu. Once a user saves the document, the label will automatically be applied.
+ 
+#### Groups & Sites 
+
+If the user opted for "Groups & Sites" under the scope configuration menu, this configuration will allow the user to define the protection setting for groups and data. This includes the level of access that internal and external users will have for O365 and Microsoft Teams groups, as well as sharing policies of SharePoint sites with external users.
+
+Once a label is configured with the above policies, it must be published. Labels can be published to all or selected users and groups. Furthermore, the following settings are available prior to publication:
+
+- Users must provide a justification to remove a label or lower its classification. From a security perspective, this setting is recommended to be on as it provides more controls to govern the data landscape and increases the effort required to bypass access controls.
+- Require users to apply a label to their emails and documents
+- Require users to apply a label to their Power BI content
+- Provide users with a link to a custom help page
+
+## Encryption
+
+Purview Information Protection uses Azure Rights Management (RMS) to encrypt sensitive files and emails. RMS uses Azure AD credentials to validate  whether the user has right access permissions to view the document. This applies to both internal and external users. 
+
+The following illustration shows the high-level encryption and decryption process for a document using RMS as provided in [Microsoft's documentation](https://learn.microsoft.com/en-us/azure/information-protection/how-does-it-work):
+
+![](../images/RMS.png)
+
+Azure RMS uses the following cryptographic controls to encrypt the data which can be seen in the [document provided here](https://learn.microsoft.com/en-us/azure/information-protection/how-does-it-work#cryptographic-controls-used-by-azure-rms-algorithms-and-key-lengths):
+
+![](../images/RMSCrypto.png)
+
+Additionally, Azure RMS supports the following security, compliance, and regulatory requirements:
```

Comment:
Although the regulatory requirements are useful for some contexts can we also expand the section a bit to discuss the available security controls such as double key encryption and the whole difference between Customer and Managed keys for encryption in the service.